rust/matches/src/lib.rs

```diff
@@ -95,6 +99,41 @@ pub mod matches {
 
     use super::*;
 
+    pub fn resize_oracle<'a, 'b, 'c, 'info>(
+        ctx: Context<'a, 'b, 'c, 'info, ResizeOracle<'info>>,
+        args: ResizeOracleArgs,
+    ) -> Result<()> {
+          
+        let win_oracle = &mut ctx.accounts.winOracle;
```

Comment:
You're not doing anything with this intermediate variable. Remove it and update the line below to:
```
let win_oracle_account = &mut ctx.accounts.win_oracle.to_account_info();
```
or
```
let win_oracle = &mut ctx.accounts.win_oracle.to_account_info();
```